snappiershot/snapshot/snapshot.py

```diff
@@ -29,15 +29,18 @@ def __init__(self, configuration: Optional[Config] = None) -> None:
         self._metadata: Optional[SnapshotMetadata] = None
         self._snapshot_file: Optional[_SnapshotFile] = None
 
-    def assert_match(self, value: Any, exact: bool = False, update: bool = False) -> bool:
-        """ Assert that the given value matches the snapshot on file
+    def assert_match(
+        self, value: Any, exact: bool = False, update: bool = False, ignore: List[str] = []
```

Comment:
Instead of using `ignore: List[str] = []` it would be best practice to use `ignore: List[str] = None` and add logic to catch the None. 

Every time `assert_match()` is called the values passed to `ignore` would be appended to the list so it would grow every time the method is called. In our case it might not matter since we create a new object for every comparison but it would be good practice to use `None` in place of `[]`. 

Background on why: https://docs.python-guide.org/writing/gotchas/

Reply:
fixed. i was ignoring the warning in pycharm to do exactly that because i didnt understand what it meant :)

[e83a66b](https://github.com/MORSECorp/snappiershot/pull/96/commits/e83a66b1eb0eb337c08b14a350a80b366cd8f371)